activerecord/lib/active_record/encryption/config.rb

```diff
@@ -42,6 +51,7 @@ def set_defaults
           self.previous_schemes = []
           self.forced_encoding_for_deterministic_encryption = Encoding::UTF_8
           self.hash_digest_class = OpenSSL::Digest::SHA1
+          self.support_previous_data_encrypted_non_deterministically_with_sha1_class_digest = false
```

Comment:
This option name feels like quite the mouthful, I think we can shorten it. How about something like `use_sha1_digest_for_non_deterministic_data`?

Reply:
I'll go with `support_sha1_digest_for_non_deterministic_data` since it will still use sha-256 at encryption time, and support both at decryption time.